adapter.go

```diff
@@ -89,6 +90,21 @@ func finalizer(a *Adapter) {
 	}
 }
 
+//Select conn according to table name（use map store name-index）
+type specificPolicy struct {
+	connnIndex int
+}
+
+func (p *specificPolicy) Resolve(connPools []gorm.ConnPool) gorm.ConnPool {
+	return connPools[p.connnIndex]
+}
+
+type DbPool struct {
+	dbMap  map[string]int
+	policy *specificPolicy
+	db     *gorm.DB
+}
```

Comment:
This struct is not necessary, it's quite complex.

Reply:
```specificPolicy ``` is not good enough.But the ```dbMap``` Can't replace in order to mapping ```connPools``` index---dbName.
https://github.com/casbin/gorm-adapter/blob/fbbb3193045a3f1bb302b3fb37fcd107a4e10db7/adapter.go#L218-L221
https://github.com/casbin/gorm-adapter/blob/fbbb3193045a3f1bb302b3fb37fcd107a4e10db7/adapter.go#L227
Do you think this can be improved？

Reply:
Wrap index and dbMap in a struct, and the struct has a method called `Switch(string)` witch could change the index. That could be better.